validator/src/main.rs

```diff
@@ -2126,11 +2131,12 @@ pub fn main() {
                     .help("Minimum time that the validator should not be leader before restarting")
             )
             .arg(
-                Arg::with_name("ignore_delinquent-stake")
-                    .short("i")
-                    .long("ignore-delinquent-stake")
-                    .takes_value(false)
-                    .help("Ignore delinquent stake threshold when exiting")
+                Arg::with_name("max_delinquent_stake")
+                    .long("max-delinquent-stake")
+                    .takes_value(true)
+                    .validator(is_parsable::<u8>)
+                    .value_name("PERCENT")
+                    .help("The maximum delinquent stake % permitted for an exit [default: 5]")
```

Comment:
```suggestion
                    .validator(solana_clap_utils:: input_validators::is_valid_percentage)
                    .default_value("5")
                    .value_name("PERCENT")
                    .help("The maximum delinquent stake % permitted for an exit")
```

Reply:
I didn't set the default and used value_t!() as this was used elsewhere (e.g. --tpu-coalesce-ms), I guess no need for the constant if it's not used anywhere else, let me adjust